### PR TITLE
Add pandadoc

### PR DESCRIPTION
**Pull request by blimmer**
I was surprised that PandaDoc doesn't support 2fa, so this PR adds it as an unsupported site.

In [this community thread](https://support.pandadoc.com/hc/en-us/community/posts/360051768714-Two-Factor-Authentication-login-to-PandaDoc), they recommend upvoting the [UserVoice request](https://pandadoc.uservoice.com/forums/926629-ideas-to-improve-pandadoc/suggestions/42446389-2fa-two-factor-authentication). I'd like to add a link to that, but the schema doesn't seem to support it. Is there a way I'm not aware of?

### CODE REVIEW

File: img/p/pandadoc.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64"><path d="M62.125 0H1.875C.84 0 0 .84 0 1.875v60.25C0 63.16.84 64 1.875 64h60.25C63.16 64 64 63.16 64 62.125V1.875C64 .84 63.16 0 62.125 0z" fill="#55b76f" fill-rule="evenodd"/><path d="M48.123 11.37V23.25c-1.92-1.525-4.35-2.436-6.992-2.436-3.737 0-7.048 1.823-9.094 4.627a11.2 11.2 0 0 0-2.157 6.625c0 .013-.005.005-.015-.015-.006 3.857-3.134 6.982-6.992 6.982-3.853 0-6.977-3.116-6.992-6.966v-.053c.014-3.85 3.14-6.966 6.992-6.966 2.62 0 4.904 1.443 6.1 3.577.142-.48.473-1.512.937-2.42.403-.788.922-1.47 1.207-1.82a11.22 11.22 0 0 0-8.245-3.597c-6.14 0-11.132 4.92-11.25 11.033h-.01v20.912h4.267V40.855c1.92 1.525 4.35 2.436 6.99 2.436 3.753 0 7.076-1.838 9.12-4.662 1.334-1.836 2.116-4.1 2.116-6.546 0-.02.012.004.032.058 0-.026-.002-.05-.002-.077 0-3.862 3.13-6.992 6.992-6.992 3.852 0 6.976 3.115 6.992 6.963v.058c-.016 3.848-3.14 6.963-6.992 6.963-2.622 0-4.906-1.444-6.103-3.58-.134.458-.47 1.528-.95 2.467-.39.765-.89 1.43-1.18 1.787a11.22 11.22 0 0 0 8.234 3.586c6.14 0 11.132-4.92 11.25-11.033h.01V11.37z" fill="#fff"/></svg>
```
Comment (phallobst) [inline img/p/pandadoc.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30"><path fill="#47b972" d="M0 0h30v30H0Z"/><path fill="#fff" d="M22 11a3.8 3.8 0 0 0-2.6-.9 4.4 4.4 0 0 0-3.4 1.5l1.2 1.4a3 3 0 0 1 2.2-1 2.8 2.8 0 0 1 2.7 3 2.8 2.8 0 0 1-2.7 3c-4.2 0-3-7.8-8.6-7.8A4.7 4.7 0 0 0 6 15v8.5h2V19a3.9 3.9 0 0 0 2.6.9 4.4 4.4 0 0 0 3.5-1.5L12.8 17a3 3 0 0 1-2.2 1 2.8 2.8 0 0 1-2.7-3 2.8 2.8 0 0 1 2.8-3c4.1 0 3 7.8 8.5 7.8a4.7 4.7 0 0 0 4.8-4.9V6.5h-2z"/></svg>
```

File: entries/p/pandadoc.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "PandaDoc": {
+    "domain": "pandadoc.com",
+    "url": "https://www.pandadoc.com/",
```
Comment (phallobst) [inline entries/p/pandadoc.com.json]:
```suggestion
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by blimmer** [comment]
Hey @phallobst, thanks for your review! If you could explain the two changes you requested I'd be happy to try to update the test suite to identify these 😄

**Comment by phallobst** [comment]
We usually leave out the `url` field if `https://<domain>` is pointing to the correct page (even if some kind of redirection is used).

The SVG logo is just an optimized version, courtesy of [svgo](https://github.com/svg/svgo), [SVGOMG](https://jakearchibald.github.io/svgomg/) and/or [nano](https://vecta.io/nano).

I'm not sure if it's feasible to adjust the test suite.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**